lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -0,0 +1,575 @@
+package LedgerSMB::Admin::Command::user;
+
+=head1 NAME
+
+LedgerSMB::Admin::Command::user - ledgersmb-admin 'user' command
+
+=cut
+
+use strict;
+use warnings;
+
+use Getopt::Long qw(GetOptionsFromArray);
+use LedgerSMB::Admin::Command;
+use LedgerSMB::App_State;
+use LedgerSMB::Company;
+use LedgerSMB::Database;
+use LedgerSMB::DBObject::User;
+use LedgerSMB::Entity::Person::Employee;
+use LedgerSMB::Entity::User;
+use LedgerSMB::PGDate;
+
+use Moose;
+extends 'LedgerSMB::Admin::Command';
+use namespace::autoclean;
+
+use Feature::Compat::Try;
+
+# LedgerSMB database interface
+has db => (is => 'rw');
```

Comment:
Are you sure this one is correct? Shouldn't it inherit it from `LedgerSMB::Admin::Command`?

Reply:
Indeed. Fixed.